s3/README.md

```diff
@@ -0,0 +1,41 @@
+Quick Start
+===============
+### 1. Set Up YCSB
+
+Clone the YCSB git repository and compile:
```

Comment:
We try to encourage folks to download binary releases now. Could you point them to https://github.com/brianfrankcooper/YCSB/releases/ instead of compiling a git checkout?